modules/extensions/src/data-filter/data-filter-extension.ts

```diff
@@ -70,6 +77,11 @@ export type DataFilterExtensionProps<DataT = any> = {
    * @default true
    */
   filterTransformColor?: boolean;
+  /**
+   * The categories which define whether an object should be rendered.
+   * @default []
+   */
+  filterCategories: [number, number] | [number, number][];
```

Comment:
Audit nit: Worth having two types or unify to always require array?

Reply:
I think the very common use case is to filter on a single value, and avoiding the nested array in this case makes the code cleaner:

```
getFilterCategory: d => d.color,
filterCategories: ['red', 'blue']
```

The nested arrays are a complication that I don't think we should force on the user unless they filtering multiple categories